### PR TITLE
Allow Port to be configured to another port than port 80

### PR DESCRIPTION
**Pull request by derqnaque**
Hello Bethesque,

I really like the pact broker, and having a docker app makes a setup really easy. Thank you for that.

We want to run the pact broker container on a host that has an apache upfront, hosting some more apps. So unfortunatly we could not use the standard port 80 for the app.

I added a little template for the nginx configuration to allow the port to be set to another port on the start of the container (Basically some simple sed magic). To run the container you need to set an Environment variable with e.g. -e PACT_BROKER_PORT=5060 and use the same port with -p 5060:5060, so the headers also dont need to be rewritten.

If you like the change, I would be delighted, if you pull it in.

Best regards, Niels

### CODE REVIEW

**Comment by bethesque** [comment]
Have not forgotten this, just need to find the time to test this appreciate your contribution.

**Comment by derqnaque** [comment]
Ok, Thanks. Didn't intend to press it either. 
We can perfectly work with our branch for now.

**Comment by elgalu** [comment]
This PR doesn't update [script/test.sh](https://github.com/DiUS/pact_broker-docker/blob/master/script/test.sh).

It doesn't seem to be backward compatible, i.e. `PACT_BROKER_PORT` doesn't default to port 80.

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<component name="InspectionProjectProfileManager">
```
Comment (bethesque) [inline .idea/inspectionProfiles/Project_Default.xml]: Was this file accidentally committed? It doesn't seem necessary for everyone else.

**Comment by bethesque** [comment]
As @elgalu suggests, can we make this default to port 80?

File: .idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (derqnaque) [inline .idea/inspectionProfiles/Project_Default.xml]: Yes it was. It should be in .gitignore. Actually it it is.
Unfortunatly this week, it's me, that doesn't have much time to fix things. 
I will try to fix async, so please keep on commenting if you'll find things, that are not ok.

**Comment by derqnaque** [comment]
yes sure, a default on port 80 is perfectly fine with me.

**Comment by bethesque** [comment]
Ta!

**Comment by elgalu** [comment]
I think we can close this PR and continue on https://github.com/DiUS/pact_broker-docker/pull/17

**Comment by bethesque** [comment]
Closing as another solution was implemented.

**Pull request closed without merging**